### ISSUE TITLE
installation instructions insufficient

### ISSUE THREAD

**Issue by jethrolarson** [ivan-kleshnin/tetris-cyclejs#6]
Tried pulling this down to play with it but webpack wouldn't load. It looks like your webpack.config is in es6 and node doesn't support es6 modules yet. I guess your environment is doing something fancy to get it to work. That's not typical though. 

I transpiled the webpack config locally to try to get it working but even then the public directories aren't getting created and filled with the appropriate files.

**Comment by ivan-kleshnin** [comment]
@jethrolarson yeah, sorry. This was a draft I started half-year ago and didn't have time to finish.
I plan to come back to this project later but can't promise.

Relative imports and ES6...
I usually use `bin/install` to update webpack and tests to run through `babel-node`.
So to run this you need 
1. to have a platform which supports symlinks (Unix-like)
2. Run `npm install` 
3. Run `bin/install`
4. make sure you use Babel5 (`npm install babel@5.?? -g`) or issue `source bin/env` to enter local shell.

Babel6 has introduced a way to alias folders so you can get away without symlinks.
But I didn't try that yet so for now the approach I described is an approach I use for all my projects.
No so fancy, but not obvious either. I need to add `bin/install` to NPM `postinstall` step.
